CRM/Civicase/Form/Report/BaseExtendedReport.php

```diff
@@ -627,7 +627,6 @@ public function buildQuery($applyLimit = TRUE) {
       if (!empty($this->_params['number_of_rows_to_render'])) {
         $this->_dashBoardRowCount = $this->_params['number_of_rows_to_render'];
       }
-      $this->limit();
```

Comment:
Looks good @deb1990 but since we are not intrested in applying a limit for pivot reports at all and do not want the code in the IF block to run we should probably fix it in this line:
https://github.com/compucorp/uk.co.compucorp.civicase/blob/a49b0f40daf1f7f932c66e72ebb75dacc25a61c3/CRM/Civicase/Form/Report/BaseExtendedReport.php#L598

and change it to
```php
public function buildQuery($applyLimit = FALSE) {
```